src/platform/qt/scripting/AutorunScriptModel.cpp

```diff
@@ -5,22 +5,58 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 #include "scripting/AutorunScriptModel.h"
 
-#include "ConfigController.h"
-#include "LogController.h"
+#include <QtDebug>
+
+QDataStream& operator<<(QDataStream& stream, const QGBA::AutorunScriptModel::ScriptInfo& object) {
+	stream << QGBA::AutorunScriptModel::ScriptInfo::VERSION;
+	stream << object.filename.toUtf8();
+	stream << object.active;
+	return stream;
+}
+
+QDataStream& operator>>(QDataStream& stream, QGBA::AutorunScriptModel::ScriptInfo& object) {
+	QByteArray versionByte = stream.device()->peek(1);
+	int version = (uint8_t) versionByte[0];
+	if (version == 0) {
+		stream >> object.filename;
```

Comment:
I'm not sure it's worth maintaining compat with a version that single digit users are using at the moment.